modules/cnf-provisioning-and-deploying-a-distributed-unit-(du)-manually.adoc

```diff
@@ -0,0 +1,130 @@
+
+A cell in the context of a DU can be seen as a real-time application performing intensive digital signal processing, data transfer,
+and algorithmic tasks.
+Cells often use hardware acceleration (FPGA, GPU, eASIC) for DSP processing offload, but there are also software-only implementations
+(FlexRAN), based on AVX-512 instructions.
+
+Running cell application on COTS hardware requires the following features to be enabled:
+
+* Real-time kernel
+* CPU isolation
+* NUMA awareness
+* HUGEPAGES memory management
+* Precision timing synchronization using PTP
+* AVX-512 instruction set (for Flexran and / or FPGA implementation)
+* Additional features depending on the RAN operator requirements
+
+Accessing hardware acceleration devices and high throughput network interface cards by virtualized software applications
+requires use of SR-IOV and Passthrough PCI device virtualization.
+
+In addition to the compute and acceleration requirements, DUs operate on multiple internal and external networks.
+
+== The manifest structure
+
+The profile is built from one cluster specific folder and one or more site-specific folders.
+This is done to address a deployment that includes remote worker nodes (several sites belonging to the same cluster).
+
+The [`cluster-config`](ran-profile/cluster-config) directory contains performance and PTP customizations based upon
+operator deployments in [`deploy`](../feature-configs/deploy) folder.
+
+The [`site.1.fqdn`](site.1.fqdn) folder contains site-specific network customizations.
+
+== Prerequisites
+
+Before installing the operators and deploying the DU, perform the following steps.
+
+. Clone the files in the `ran-profile` folder here:
+link:https://github.com/openshift-kni/cnf-features-deploy/tree/master/feature-configs/cn-ran-overlays/ran-profile[ran-profile].
+
+. Create a machine config pool for the RAN worker nodes. For example:
++
+[source,yaml]
+----
+cat <<EOF | oc apply -f -
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfigPool
+metadata:
+  name: worker-cnf
+  labels:
+    machineconfiguration.openshift.io/role: worker-cnf
+spec:
+  machineConfigSelector:
+    matchExpressions:
+      - {
+          key: machineconfiguration.openshift.io/role,
+          operator: In,
+          values: [worker-cnf, worker],
+        }
+  paused: false
+  nodeSelector:
+    matchLabels:
+      node-role.kubernetes.io/worker-cnf: ""
+----
+
+. Include the worker node in the above machine config pool by labelling it with the `node-role.kubernetes.io/worker-cnf` label:
++
+----
+oc label --overwrite node/<your node name> node-role.kubernetes.io/worker-cnf=""
+----
+
+. Label the node as ptp slave (DU only):
++
+----
+oc label --overwrite node/<your node name> ptp/slave=""
+----
+
+An example of labeling the nodes used in CI/CD can be seen here:
+link:https://raw.githubusercontent.com/openshift-kni/cnf-features-deploy/master/hack/setup-test-cluster.sh[setup-test-cluster.sh].
```

Comment:
Why would we not put these examples into this docs set. Why leave these examples in an upstream repository? 

We also can't distribute code/scripts via non RCM released and tracked sources for security and licencing issues.